src/dotty/tools/dotc/repl/InteractiveReader.scala

```diff
@@ -13,14 +13,13 @@ trait InteractiveReader {
 /** The current Scala REPL know how to do this flexibly.
  */
 object InteractiveReader {
-  /** Create an interactive reader.  Uses JLine if the
+  /** Create an interactive reader. Uses Ammonite if the
    *  library is available, but otherwise uses a
    *  SimpleReader. */
```

Comment:
I agree that this try-catch can be removed.

Or, are you suggesting that removing SimpleReader entirely from Dotty?